### ISSUE TITLE
Figure 2.13 on p38 of 3rd edition

### ISSUE THREAD

**Issue by ronnie-canopy** [fchollet/deep-learning-with-python-notebooks#268]
Should the action of relu on final image of the transformed letter k be only in the top right quarantine? I'd expect the component wise relu to kill the part in the negative y-axis in addition the killing the negative x-axis.

**Comment by stefdesabbata** [comment]
I noticed the same issue, and that seems to be indeed the case when testing it in code (see [full notebook here](https://colab.research.google.com/drive/1cXhIg11uInTzpvYBQsjv3mQTzlUq9g1-?usp=sharing))

```python
affine_matrix = tf.constant([
    [1,  -0.3],
    [0.5, 0.5]
    ], dtype=tf.float32)
bias = tf.constant([[-200, 50]], dtype=tf.float32)

affine_points = tf.matmul(points_tensor, affine_matrix) + bias

relu_points = keras.activations.relu(affine_points)
```

![Image](https://github.com/user-attachments/assets/c4268b56-f862-4a11-8217-affcdee81d51)
